css/_base.scss

```diff
@@ -162,6 +162,15 @@ input, button, select, textarea {
   cursor: move;
 }
 
+.favourite-button {
```

Comment:
Let's create a `c-favourite-button` instead of puting this here (_base.scss)... What do you think?